modules/irmaseal-pkg-server.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.irmaseal-pkg;
+in
+{
+
+  options.services.irmaseal-pkg = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        This option enables the IRMAseal-PKG server.
+      '';
+    };
+
+    package = mkOption {
```

Comment:
this option usually only makes sense when there are several packages available. (if there are different versions or if you can override some features). I think in our case, there is only a single package right ? Nothing to override ? (just making sure).

Reply:
Yeah for now there is only one package. I had assumed this was a good practice, but we can remove this.